packages/combobox/src/index.tsx

```diff
@@ -1148,6 +1147,17 @@ const makeHash = (str: string) => {
   return hash;
 };
 
+/**
+ * Escape regexp special characters in `str`.
+ *
+ * @param {String} str
+ * @return {String}
+ */
+
+export function escapeRegexp(str: string) {
+  return String(str).replace(/([.*+?=^!:${}()|[\]/\\])/g, "\\$1");
+}
```

Comment:
[Taken directly from the external package source](https://github.com/component/escape-regexp/blob/5ce923c1510c9802b3da972c90b6861dd2829b6b/index.js#L10-L12)